lms/djangoapps/hera/admin.py

```diff
@@ -97,6 +99,28 @@ def has_add_permission(self, request):
         return not self.model.objects.exists()
 
 
+class ScaffoldForm(ModelForm):
+    class Meta:
+        model = Scaffold
+        fields = '__all__'
+        widgets = {
+            'rephrase_color': TextInput(attrs={'type': 'color'}),
```

Comment:
should be checked whether it works in all browsers.

Reply:
works in safari, chrome and firefox.